docs/docs/deploying-to-now.md

```diff
@@ -23,3 +23,17 @@ In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can
 ```
 
 4.  Run `now` at the root of your Gatsby project, this will upload your project, run the `build` script, and then your `start` script.
+
+Note: The following steps assume that you have `now` installed on your machine.
```

Comment:
Rather than duplicate, maybe we just provide _some_ info and then a blurb like `to read more about Now deployments, check out [this document](/link/here)`

Reply:
Sorry, I'm not sure if I understand what you mean here. Are you talking about the fact that the same content is duped in two separate pages?

Reply:
Yes! I mean in the other document, maybe link to _this_ document?

Reply:
I think it's fair to treat more general purpose documents as "shells" that link to further references.

Reply:
Haha okay, I get what you mean now! I'll update it and ping you once again. Check back whenever you have time tho, no pressure. Thanks for all the feedback so far!